plugins/babel/src/run-babel.ts

```diff
@@ -38,7 +37,9 @@ export async function runBabel(
         error.details = `the problematic file was ${file}`
         throw error
       }
-      await fs.writeFile(path.join(outputPath, path.relative(base, file)), transformed.code)
+      const relativePath = path.relative(base, file)
+      await fs.mkdir(path.join(outputPath, path.dirname(relativePath)), { recursive: true })
+      await fs.writeFile(path.join(outputPath, relativePath), transformed.code)
     })
```

Comment:
From the example given in the PR description (for the first promise for `src/clients/cloudwatch.js`):
- `base` = `src`
- `file` = `src/clients/cloudwatch.js`
- `relativePath` = `clients/cloudwatch.js`
- `outputPath` = `lib`
- `path.dirname(relativePath)` = `clients`
- `path.join(outputPath, path.dirname(relativePath)` = `lib/clients`

Reply:
N.B.
```js
await fs.mkdir(path.join(outputPath, relativePath), { recursive: true })
```

Using the above line of code rather than the one used in the PR will result in ``illegal operation on a directory, open `lib/clients/cloudwatch.js` ``

Reply:
**suggestion:** Nice! Thanks Andy, what do you think about adding a comment here to clarify what these couple of lines are doing? Something like
```
// create parent directories if they don't exist before creating child file of transpiled code
```

Reply:
**suggestion**: it might a bit more readable like this?

```ts
const filePath = path.join(outputPath, path.relative(base, file))
await fs.mkdir(path.dirname(filePath), { recursive: true })
await fs.writeFile(filePath, transformed.code)
```